vectorstores/milvus/milvus.go

```diff
@@ -219,12 +219,23 @@ func (s Store) AddDocuments(ctx context.Context, docs []schema.Document,
 	textCol := entity.NewColumnVarChar(s.textField, texts)
 	metaCol := entity.NewColumnVarChar(s.metaField, metadatas)
 	vectorCol := entity.NewColumnFloatVector(s.vectorField, len(vectors[0]), vectors)
-	_, err = s.client.Insert(ctx, s.collectionName, s.partitionName, vectorCol, metaCol, textCol)
+	column, err := s.client.Insert(ctx, s.collectionName, s.partitionName, vectorCol, metaCol, textCol)
 	if err != nil {
 		return nil, err
 	}
 
-	return nil, nil
+	col, ok := column.(*entity.ColumnInt64)
+	if !ok {
+		return nil, nil
```

Comment:
if we decide to not consider this an error condition there should be a comment describing the rationale

Reply:
> if we decide to not consider this an error condition there should be a comment describing the rationale
Maybe it should return an error.